core/frontend/src/render/webgl/GLTimer.ts

```diff
@@ -8,6 +8,7 @@ import { IModelError } from "@bentley/imodeljs-common";
 import { GLTimerResult, GLTimerResultCallback } from "../RenderSystem";
 import { System } from "./System";
 
+type WebGLObject = WebGLBuffer | WebGLFramebuffer | WebGLProgram | WebGLQuery | WebGLRenderbuffer | WebGLSampler | WebGLShader | WebGLSync | WebGLTexture | WebGLTransformFeedback | WebGLVertexArrayObject | WebGLVertexArrayObjectOES;

```

Comment:
WebGLObject is only used in this file and as far as I can tell at a glance, it's not actually useful. @mgooding?

Reply:
Talked to @mgooding - I'm going to just remove the type alias and replace `WebGLObject` with `WebGLQuery`.

Reply:
I talked to @wgoehrig... he's going to replace WebGLObject with WebGLQuery. If it's a WebGL 1 context, it's actually WebGLQueryEXT but it's safe to use in the same places with a WebGL 1 Context.

I don't know why I typed it this way originally... I assume I was going off of the native API at the time due to a lack of documentation and just wanted to avoid an `any` type?